app/_gateway_entities/rbac.md

```diff
@@ -152,58 +153,68 @@ curl -Ls get.konghq.com/quickstart | bash -s -- -e "KONG_LICENSE_DATA" \
    -e "KONG_ENFORCE_RBAC=on" \
    -e "KONG_ADMIN_GUI_AUTH=basic-auth" \
    -e "KONG_PASSWORD=kong" \
-   -e "KONG_PORTAL=on" \
    -e 'KONG_ADMIN_GUI_SESSION_CONF={"secret":"kong", "cookie_lifetime":300000, "cookie_renew":200000, "cookie_name":"kong_cookie", "cookie_secure":false, "cookie_samesite": "off"}'
 ```
 
 {% endnavtab %}
-{% navtab "Kong Gateway" %}
-
-1. Before enforcing RBAC on your {{site.base_gateway}} instance, we recommend that you create a `super-admin` user first: 
+{% navtab "Docker container" %}
 
-```sh
- curl -i -X POST http://localhost:8001/rbac/users \
-   -H 'Kong-Admin-Token:$YOUR_TOKEN' \
-   --data name=super-admin \
-   --data user_token=$SUPER_ADMIN_USER_TOKEN
-```
-Creating the `super-admin` username automatically adds the user to the `super-admin` role.
+If you have a running {{site.base_gateway}} Docker container, you can enable RBAC using environment variables.
```

Comment:
It doesn't have to be a docker container, the env variables can be used in any on-prem deployment.